README.md

```diff
@@ -1,24 +1,215 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-Things you may want to cover:
+Modify this file to match your project and remove sections that don't apply.
 
-* Ruby version
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-* System dependencies
+OPTIONAL SECTIONS:
+- FAQ
 
-* Configuration
+After you're finished please remove all the comments and instructions!
+-->
 
-* Database creation
+<!-- TABLE OF CONTENTS -->
 
-* Database initialization
+# 📗 Table of Contents
 
-* How to run the test suite
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [📷 Video Recording](#video-record)
```

Comment:
- [ ] The video recording must include a piece of code or feature that you are most proud of. This is present in minimum project requirements.